### PR TITLE
Added Resizr

### PR DESCRIPTION
**Pull request by onurgenes**
## Project URL
https://github.com/onurgenes/Resizr

## Category
iOS/MacOS

## Description
Resizr is a MacOS application for creating AppIcon for iOS and Android (coming soon) apps.
 
## Checklist

- [x] Edit [applications.json](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/applications.json) instead of [README.md](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/README.md).
- [x] Only one project/change is in this pull request
- [x] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,6 +230,7 @@ You can see in which language an app is written. Currently there are following l
 - [Localizations](https://github.com/athiercelin/Localizations) - Localizations is an macOS app that manages your Xcode project localization files (.strings).  ![swift_icon] 
 - [Menubar Colors](https://github.com/nvzqz/Menubar-Colors) - macOS app for convenient access to the system color panel.  ![swift_icon]  <details> <summary> Screenshots </summary> <p float="left">  <bt><img src='https://raw.githubusercontent.com/nvzqz/Menubar-Colors/master/./Example.jpg' width="400"/>  </p></details> 
 - [MyTouchbarMyRules](https://github.com/toxblh/MTMR) - App to customize your Touch Bar as you want.  ![swift_icon] 
+- [Resizr](https://github.com/onurgenes/Resizr) - MacOS application for creating AppIcon for iOS and Android apps.  ![swift_icon]
```
Comment (serhii-londar) [inline README.md]: @onurgenes Please update applications.json.

**Comment by onurgenes** [comment]
Changed order and added to applications.json 👍🏻

**Comment by onurgenes** [comment]
Any updates on this?

**Comment by serhii-londar** [comment]
@onurgenes Hi. I was working on CI fixes. Thank for your contribution! 👍

**Pull request merged**